tests/datasource/fluent/test_pandas_azure_blob_storage_datasource.py

```diff
@@ -282,91 +282,6 @@ def test_construct_csv_asset_directly(
     assert m1 is not None
 
 
-@pytest.mark.integration
```

Comment:
All of the tests that have been removed were testing inherited behavior on the same `CSVAsset`.